src/attributes.md

```diff
@@ -184,11 +184,17 @@ macro scope.
   object file that this item's contents will be placed into.
 - `no_mangle` - on any item, do not apply the standard name mangling. Set the
   symbol for this item to its identifier.
-- `doc` - Doc comments such as `/// foo` are equivalent to `#[doc = "foo"]`.
 - `must_use` - on structs and enums, will warn if a value of this type isn't used or
    assigned to a variable. You may also include an optional message by using
    `#[must_use = "message"]` which will be given alongside the warning.
 
+### Documentation
+
+The `doc` attribute is used to document items and fields. [Doc comments]
```

Comment:
Is this supposed to be a link anchor? I can't find its reference link.

Reply:
Yes. Forgot to add it. Added and rebased.